lib/Core/Repository/Aggregate/Repository.php

```diff
@@ -38,10 +38,23 @@ class Repository implements RepositoryInterface
      */
     public function __construct(
         RepositoryInterface $ezRepository,
-        array $customRepositories
+        array $customRepositories = []
```

Comment:
Do we now need a second param here?

Reply:
We don't use it here, I just always add it to aggregate implementation of something.

Reply:
Then you don't need `addRepository` method, since you can use `$definition->replaceArgument(1, [...])` in the compiler pass, where `[...]` is an array of `Reference` objects.

Reply:
Agreed, updated in https://github.com/netgen/ezplatform-site-api/pull/67/commits/45911390a2a98b6d74383bf0dd57435aba957503